docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -209,8 +213,26 @@ Release
       git tag --annotate "${TAG}" "${BRANCH}" -m "Tag version ${TAG}"
       git push origin "${TAG}"
 
+#. Go to `Jenkins`_ and force a build on the release branch to test the latest commit.
+
+   Currently, jobs cannot be created for git tags so the latest commit must be test instead.
+   This must be the same commit as the tag.
+   The git commit that was used can be seen on the summary page for any build.
+
+   If `FLOC-3917 <https://clusterhq.atlassian.net/browse/FLOC-3917>`_ has not been resolved, then the "_build_vagrant_basebox_for_flocker_tutorial" job must also be run.
```

Comment:
Put this in double-backquotes and you should be able to remove the non-word "basebox" from the spelling list.